tests/core/policies/test_ted_policy.py

```diff
@@ -153,6 +158,35 @@ def test_doesnt_checkpoint_with_zero_eval_num_examples(self, tmp_path: Path):
         assert not checkpoint_dir.is_dir()
         assert len([w for w in warning if warn_text in str(w.message)]) == 1
 
+    @pytest.mark.parametrize(
+        "should_finetune, epoch_override, expected_epoch_value",
+        [
+            (True, 2, 2),
+            (True, 1, 1),
+            (
+                False,
+                20,
+                TEDPolicy.defaults[EPOCHS],
+            ),  # trained_policy uses default epochs during training
+            (False, TEDPolicy.defaults[EPOCHS], TEDPolicy.defaults[EPOCHS]),
```

Comment:
I wanted to have a test case with the default value of `epoch_override` in the `load` function, i.e. `1`. I've now changed it to `TEDPolicy.defaults[EPOCHS]`.
`2` was just an arbitrary number other than the default but I've now changed it to `TEDPolicy.defaults[EPOCHS] + 1`

Do you think the extra test case with the default value of the function is unnecessary?

Reply:
- I think we could remove:  `(False, TEDPolicy.defaults[EPOCHS], TEDPolicy.defaults[EPOCHS],),` because I'm not sure what this would test and because `(False, TEDPolicy.defaults[EPOCHS]+1, TEDPolicy.defaults[EPOCHS],),` already tests that the override value is ignored.
- And I think we could remove `(True, TEDPolicy.defaults[EPOCHS], TEDPolicy.defaults[EPOCHS]),` because it would just test that the epoch is overwritten with the (identical) epoch_overwrite value (so it would also pass if the function did not override anything :D) while `(True, TEDPolicy.defaults[EPOCHS] + 1, TEDPolicy.defaults[EPOCHS] + 1),` tests exactly that, no?
- Regarding the default values, I think that the above should suffice as test (we can't test all n \in N here and 1/2 are/should be no special cases really regarding the overwrite), no?

Reply:
You've convinced me. Removed 👍